src/workerd/server/workerd.capnp

```diff
@@ -601,6 +601,17 @@ struct Worker {
     # workerd uses SQLite to back all Durable Objects, but the SQL API is hidden by default to
     # emulate behavior of traditional DO namespaces on Cloudflare that aren't SQLite-backed. This
     # flag should be enabled when testing code that will run on a SQLite-backed namespace.
+
+    container @5 :ContainerOptions;
+    # Optional configuration options for local development container interaction.
+
+    struct ContainerOptions {
+      containerName @0 :Text;
+      # Name of the container that should be created.
+
+      imageTag @1 :Text;
```

Comment:
aren't tag local to images (i.e. 2 images can have the same tag).
If that's true the tag is probably not enough?

Reply:
By default we use latest of an image. If we want to support it in the future, we can definitely support different tags

Reply:
Sounds like this field is actually the image name?

Reply:
yeah imageName might be a better name, agreed.